src/routes/products/ProductsPage.tsx

```diff
@@ -0,0 +1,187 @@
+    ],
+    imageUrl:
+      "https://www.sbs.com.au/food/sites/sbs.com.au.food/files/styles/full/public/gettyimages-169371459.jpg?itok=eZUqM9o2",
+  },
+  {
+    name: "Mega Strength",
+    activeIngredients: ["Salivarius", "Lactobacillius", "Vitamin C"],
+    inActiveIngredients: [
+      "purified talc",
+      "stearic acid",
+      "povidone",
+      "starch pregelatinised",
+    ],
+    imageUrl:
+      "https://www.sbs.com.au/food/sites/sbs.com.au.food/files/styles/full/public/gettyimages-169371459.jpg?itok=eZUqM9o2",
+  },
+  {
+    name: "Pre Natal Complex",
+    activeIngredients: ["Salivarius", "Lactobacillius", "Vitamin C"],
+    inActiveIngredients: [
+      "purified talc",
+      "stearic acid",
+      "povidone",
+      "starch pregelatinised",
+    ],
+    imageUrl:
+      "https://www.sbs.com.au/food/sites/sbs.com.au.food/files/styles/full/public/gettyimages-169371459.jpg?itok=eZUqM9o2",
+  },
+  {
+    name: "Night Nurse Cold & Flu",
+    activeIngredients: ["Salivarius", "Lactobacillius", "Vitamin C"],
+    inActiveIngredients: [
+      "purified talc",
+      "stearic acid",
+      "povidone",
+      "starch pregelatinised",
+    ],
+    imageUrl:
+      "https://www.sbs.com.au/food/sites/sbs.com.au.food/files/styles/full/public/gettyimages-169371459.jpg?itok=eZUqM9o2",
+  },
+];
+
+const PageWrapper = styled.div`
+  display: flex;
+  padding: var(--spacing-md);
+`;
+
+const ProductsWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-left: var(--spacing-md);
+  width: 100%;
+`;
+
+const ProductsPage = () => {
+  const [selectedFilters, setSelectedFilters] = useState<Array<string> | null>(
+    null
+  );
+  const [productFilterApiParams, setProductFilterApiParams] = useState({});
+  const [isMobileDrawOpen, setIsMobileDrawOpen] = useState(false);
+  const [screenSize, setScreenSize] = useState<WindowSizeEnum>(
+    window.innerWidth > Breakpoints.md
+      ? WindowSizeEnum.LARGE
+      : WindowSizeEnum.SMALL
+  );
+
+  const updateFilters = (newFilter: string) => {
+    setSelectedFilters([...(selectedFilters || []), newFilter]);
+  };
+
+  const clearFilters = () => {
+    setSelectedFilters(null);
+    setProductFilterApiParams({});
+  };
+
+  const toggleMobileFilters = () => {
+    setIsMobileDrawOpen(!isMobileDrawOpen);
+  };
+  const setSize = useCallback((dimensions: Dimensions) => {
+    if (dimensions.width > Breakpoints.md) {
+      setScreenSize(WindowSizeEnum.LARGE);
+    } else {
+      setScreenSize(WindowSizeEnum.SMALL);
+    }
+  }, []);
+
+  useWindowResize(setSize);
+
+  // Note for testing purposes the below function will only work on the following filters atm: activeIngredients, dietaryRequirements
+  // Its set up to reconstruct the API url with applied params, I admit its not the cleanest code in the world
+
+  // If API accepts null for query params (ie activeIngredients=[]) we could probably simplfy this code
+
+  const constructApiFilterString = (
```

Comment:
This doesn't quite work yet.
If I add the filters
Name: 'Meg'
Active ingredients: Vitamin C
Dietary requirements: Snapdragon, Vitamin C

The constructed URL should be:
`/products?name=Meg&activeIngredients[]=Vitamin C&dietaryRequirements[]=Snapdragon&dietaryRequirements[]=Vitamin C`

At the moment it gives:
`/products?activeIngredients[]=Vitamin C&dietaryRequirements[]=,Snapdragon,Vitamin C`

So

- Missing 'name'
- Extra leading comma (but I don't think we need commas at all anyway)
- Each dietaryRequirements needs to be split up

Sorry if that wasn't clear enough from my examples!